webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -55,6 +57,7 @@ const WithTooltip = ({
             onExit={onExit}
             shouldUpdatePosition={shouldUpdatePosition}
             disabled={disabled}
+            onEntering={onEntering}
```

Comment:
lets not add another prop to this component

Reply:
The extra prop has been removed, here is a screenshot of the UI change

![image](https://github.com/user-attachments/assets/892c2ad6-322d-4399-a8ed-fa88c1df49f7)